concordia/views.py

```diff
@@ -202,59 +216,22 @@ def get_template_names(self):
 class CollectionView(TemplateView):
     template_name = "transcriptions/create.html"
 
+    @transaction.non_atomic_requests
```

Comment:
This could use a comment of some sort – since the intention is to disable automatic transactions I was expecting to see transaction handling in the code but did not.

Reply:
Regarding @transaction.non_atomic_requests:
1. This will not rollback any changes made to database, will not behave as atomic_request, which will treat each view as a transaction
2. To not to rollback everything when there is an error in processing images in task. This is for initial testing until we handle all error scenarios.

Reply:
As discussed in the meeting, let's create a separate issue for this independent of this PR to discuss error reporting & handling. I would vote for removing `non_atomic_requests` now so it will follow the default behaviour until that strategy is implemented.

Reply:
Sure, will create a separate issue.